packages/cmf/scripts/cmf-settings.merge.js

```diff
@@ -145,10 +150,8 @@ function merge(options, errorCallback) {
 		file.write(JSON.stringify(settingWithoutI18n) + String.fromCharCode(10));
 		file.end();
 		logger('CMF settings has been merged');
-		return jsonFiles.concat(cmfconfigPath);
```

Comment:
in term of responsabilities I have moved this into the webpack plugin because it override the config, so he knows where is the file.

Reply:
ok, understand.
thanks for help!